lib/install/management/store_files.go

```diff
@@ -83,7 +88,7 @@ func (d *Dispatcher) deleteImages(conf *config.VirtualContainerHostConfigSpec) e
 			continue
 		}
 
-		if len(children) == 0 {
+		if len(children) == 1 { // when image store path is given, we need to delete VIC dir too when it is empty.
```

Comment:
What does children stand for? If it is the number of child folders, is it possible to have any other folders retain here? Or is it thread safe when you delete 2 VCH at the same time.

Reply:
Yes, it stands for folders and files under a root directory. The parallel file access is protected by VC itself.